### PR TITLE
Clarify datadog rate limit

### PR DESCRIPTION
**Pull request by arapulido**
Selecting a small time window for Datadog, can cause returning an empty series.
Trying to discourage this by avoiding reducing the window in the example in the docs.

Also, added some clarification about Datadog's rate limits

### Checklist

- [X] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.6/scalers/datadog.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ You should use `TriggerAuthentication` CRD to configure the authentication:
 
 ### Example
 
+The example below uses the default KEDA polling interval (30 seconds). Take into account that [API Datadog endpoints are rate limited](https://docs.datadoghq.com/api/latest/rate-limits/) and reducing the polling interval can accelerate reaching it. If your account has reached its rate limit, a relevant error will be logged in KEDA.
```
Comment (tomkerkhove) [inline content/docs/2.6/scalers/datadog.md:44]: Might be good to make this a more prominent piece by moving it to the top of the article?

For example:
> :lightbulb: Consider the [API Datadog endpoints are rate limited](https://docs.datadoghq.com/api/latest/rate-limits/) when defining polling interval

**Review by tomkerkhove — APPROVED** [review]
Can you update v2.7 doc as well please?

File: content/docs/2.6/scalers/datadog.md (hunk 1 above)
Comment (arapulido) [reply content/docs/2.6/scalers/datadog.md:46]: done

**Comment by arapulido** [comment]
> Can you update v2.7 doc as well please?
done. thanks!

**Review by JorTurFer — APPROVED** [review]

**Comment by JorTurFer** [comment]
Thanks for the improvement!

**Review by tomkerkhove — APPROVED** [review]
Thank you @arapulido!

**Pull request merged**